### DISCUSSION TITLE
Nebula Graph Database

### DISCUSSION THREAD

**Discussion by ajeetraina** [docker/extension-ideas#143] [Ideas]
Note: This is a transfer from our initial method of collecting new extension requests. If you have additional information on how this will help you or other functionality you'd like to see, please add more details in the comments.

https://www.nebula-graph.io/

**Comment by ajeetraina** [comment]
Good to see that Nebula Graph team built this extension. Tested it and it worked like a charm. 

https://github.com/wey-gu/nebulagraph-docker-ext

**Comment by wey-gu** [comment]
Thanks @ajeetraina for guiding us here, it brought NebulaGraph user/dev accessing a learning/testing/dev env easily like never before.

We could literally have a whole cluster up and running from scratch in one click.

Try from here before it's approved in marketplace:)

https://open.docker.com/extensions/marketplace?extensionId=weygu/nebulagraph-dd-ext&tag=latest
